packages/curriculum-compiler-json/__tests__/fixtures/glossary/sample/parsed.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rawText": "# Markdown Formatting\n\nJust as we use punctuation to improve readability, we can use markdown to format text in our comment section.\n\nBefore you leave your first submission, it might help to brush up your markdown.\n\nWhen submitting an answer in the comment section, you'll want to use something like this:\n\n```py\nprint(\"So this is markdown!\");\nthis should \n```\n\nThat is called a code block. To write it, you start with ```` ``` ```` and end with ```` ``` ````, just like this:\n\n````md\n```python\n# your code\n# goes here\n```\n````\n\nNotice how we also include `python` after the first ```` ``` ````. This enables syntax highlighting, which is a way to colour different parts of the code to make it more readable.\n\nDepending on what you are learning, you can use the following tags: `python`, `javascript`, `sql`, `html`, `css`, `java`, `go`, `bash`, `git`, `docker`, `jsx` or `plain-text`.\n\nWhen you're leaving a comment, you can press the `M⬇` button for a markdown cheat-sheet.\n\nYou can also press the eye button to preview your comment.\n\n![markdown-and-preview-button-positions](https://img.enkipro.com/a92eef7bc6e5ad805e04d5c1e240924b.gif)\n",
```

Comment:
same thing here, removing the extra `this should`
```suggestion
  "rawText": "# Markdown Formatting\n\nJust as we use punctuation to improve readability, we can use markdown to format text in our comment section.\n\nBefore you leave your first submission, it might help to brush up your markdown.\n\nWhen submitting an answer in the comment section, you'll want to use something like this:\n\n```py\nprint(\"So this is markdown!\");\n```\n\nThat is called a code block. To write it, you start with ```` ``` ```` and end with ```` ``` ````, just like this:\n\n````md\n```python\n# your code\n# goes here\n```\n````\n\nNotice how we also include `python` after the first ```` ``` ````. This enables syntax highlighting, which is a way to colour different parts of the code to make it more readable.\n\nDepending on what you are learning, you can use the following tags: `python`, `javascript`, `sql`, `html`, `css`, `java`, `go`, `bash`, `git`, `docker`, `jsx` or `plain-text`.\n\nWhen you're leaving a comment, you can press the `M⬇` button for a markdown cheat-sheet.\n\nYou can also press the eye button to preview your comment.\n\n![markdown-and-preview-button-positions](https://img.enkipro.com/a92eef7bc6e5ad805e04d5c1e240924b.gif)\n",
```

Reply:
again, test only, shouldnt really matter